framework/source/class/qx/dev/unit/MRequirements.js

```diff
@@ -256,6 +256,18 @@ qx.Mixin.define("qx.dev.unit.MRequirements", {
     },
 
 
+    /**
+     * Checks if the application is running on Windows 10
+     *
+     * @return {Boolean} <code>false</code> if operating system is Windows 10
+     */
+    hasNoWin10 : function()
+    {
+      var isWin10 = (qx.core.Environment.get("os.name") === "win" && qx.core.Environment.get("os.version") === "10");
+      return (isWin10 ? false : true);
```

Comment:
`isWin10` is already a boolean. It doesn't need the trinary expression. Could just `return ! isWin10;`

Reply:
@derrell please view the other methods in the file. they are all written similar e.g the method `hasNowWin7`.